politeiawww/api/cms/v1/v1.go

```diff
@@ -146,8 +148,8 @@ type LineItemsInput struct {
 	Subtype       string        `json:"subtype"`       // Subtype of work performed
 	Description   string        `json:"description"`   // Description of work performed
 	ProposalToken string        `json:"proposaltoken"` // Link to politeia proposal that work is associated with
-	Hours         float64       `json:"hours"`         // Number of Hours (if labor)
-	TotalCost     float64       `json:"totalcost"`     // Total USD cost of line item (if expense or misc)
+	Minutes       uint16        `json:"minutes"`       // Number of minutes (if labor)
+	TotalCost     uint16        `json:"totalcost"`     // Total cost (in USD cents) of line item (if expense or misc)
```

Comment:
Same here, just use uint. This one in particular makes no sense because there certainly can be a line item that is more expensive than $6553.35